jrnl/args.py

```diff
@@ -17,6 +17,48 @@
 from .plugins import util
 
 
+def deserialize_config_args(input: list) -> dict:
+
+    """
+
+    Convert a two-element list of configuration key-value pair into a flat dict
+
+    :param input: list of configuration keys in dot-notation and their respective values.
+    :type input: list
+    :return: A single level dict of the configuration keys in dot-notation and their respective desired values
+    :rtype: dict
+    """
+
+    assert len(input) == 2
+    runtime_modifications = {}
+
+    cfg_key = input[0]
+    cfg_value = input[1]
+    cfg_value = cfg_value.strip()
+
+    # Convert numbers and booleans
+    if cfg_value.isdigit():
+        cfg_value = int(cfg_value)
+    elif cfg_value.lower() == "true":
+        cfg_value = True
+    elif cfg_value.lower() == "false":
+        cfg_value = False
+
+    runtime_modifications[cfg_key] = cfg_value
```

Comment:
Does this handle keys with dots?

Reply:
Yes, it stores the keys in dot notation. Those are unpacked into the config tree in apply_overrides